### ISSUE TITLE
Windows title action hover is rounded

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#500]
Accidental?

![image](https://user-images.githubusercontent.com/22350/169833384-0992ea10-136b-432d-9320-8a8457b8a4a8.png)

**Comment by vscodenpa** [comment]
confidence for label accessibility: 0.5. does not meet threshold

**Comment by vscodenpa** [comment]
confidence for assignee sbatten: 0.75. does meet threshold

**Comment by vscodenpa** [comment]
confidence for label accessibility: 0.5. does not meet threshold

**Comment by vscodenpa** [comment]
confidence for assignee sbatten: 0.75. does meet threshold

**Comment by vscodenpa** [comment]
confidence for label accessibility: 0.5. does not meet threshold

**Comment by vscodenpa** [comment]
confidence for assignee sbatten: 0.75. does meet threshold

**Comment by vscodenpa** [comment]
confidence for label accessibility: 0.5. does not meet threshold

**Comment by vscodenpa** [comment]
confidence for assignee sbatten: 0.75. does meet threshold

**Comment by vscodenpa** [comment]
confidence for label accessibility: 0.5. does not meet threshold

**Comment by vscodenpa** [comment]
confidence for assignee sbatten: 0.75. does meet threshold

**Comment by vscodenpa** [comment]
confidence for label accessibility: 0.5. does not meet threshold

**Comment by vscodenpa** [comment]
confidence for assignee sbatten: 0.75. does meet threshold

**Comment by vscodenpa** [comment]
confidence for label accessibility: 0.5. does not meet threshold

**Comment by vscodenpa** [comment]
confidence for assignee sbatten: 0.75. does meet threshold
